src/cron.c

```diff
@@ -46,7 +46,7 @@
 #endif
 
 #ifdef HAVE_SYS_FCNTL_H
-# include <sys/fcntl.h>
+# include <fcntl.h>
```

Comment:
The ifdef has to be modified as well. Or just include fcntl.h unconditionally.

Reply:
So yeah. Just drop the ifdef and move the fcntl.h include before the other conditional include for tidiness.